app/views/mybookings/backend/resource_types/_form_fields.html.haml

```diff
@@ -25,6 +25,8 @@
         = f.input :limit_days_for_recurring_events, min_max: true, hint: t('mybookings.backend.resource_types.hints.limit_days_for_recurring_events')
       .col-xs-12.col-sm-4
         = f.input :limit_days_for_feedback, min_max: true, hint: t('mybookings.backend.resource_types.hints.limit_days_for_feedback')
+      .col-xs-12.col-sm-4
```

Comment:
Use the twelve columns better.

![captura de pantalla 2018-03-16 a las 11 42 43](https://user-images.githubusercontent.com/454382/37516733-6fba4492-290f-11e8-95fb-3c9f76162beb.png)

Reply:
Ok.